code/controllers/subsystem/SSticker.dm

```diff
@@ -607,6 +607,12 @@ SUBSYSTEM_DEF(ticker)
 			var/mob/M = m
 			H.add_hud_to(M)
 
+	// Tell people how many kudos they got this round
+	// Besides, what's another loop over the /entire player list/
+	for(var/mob/M in GLOB.player_list)
+		if(length(M.mind?.kudos_recieved_from))
+			to_chat(M, "<span class='green big'>You recieved [length(M.mind?.kudos_recieved_from)] kudos this round! Keep up the good work!</span>")
```

Comment:
```suggestion
			to_chat(M, "<span class='green big'>You received [length(M.mind?.kudos_recieved_from)] kudos this round! Keep up the good work!</span>")
```
Theres the "kudos_recieved_from" too.

Reply:
Agh, my biggest weakness. Spelling `received` correctly. Thank you 😅